src/microbe_stage/editor/MicrobeEditorWorld.tscn

```diff
@@ -27,12 +25,7 @@ shader_param/maskTexture = ExtResource( 3 )
 
 [node name="MicrobeEditorWorld" type="Node"]
 
-[node name="PrimaryCamera" type="Camera" parent="." instance=ExtResource( 5 )]
```

Comment:
These look very much incorrect changes that should not be in this PR, please remove these, otherwise this can't be merged without breaking existing features.